opencga-server/src/main/java/org/opencb/opencga/server/generator/writers/cli/ExecutorsCliRestApiWriter.java

```diff
@@ -271,97 +282,129 @@ private String getJavaMethodName(CategoryConfig config, String commandName) {
+    private String getBodyParams(RestCategory restCategory, RestEndpoint restEndpoint, CategoryConfig config, String commandName) {
         StringBuilder sb = new StringBuilder();
         String bodyClassName = restEndpoint.getBodyClassName();
         if (StringUtils.isNotEmpty(bodyClassName)) {
-            for (RestParameter restParameter : restEndpoint.getParameters()) {
-                if (restParameter.getData() != null && !restParameter.getData().isEmpty()) {
-                    sb.append(generateBeans(restParameter.getData()));
-                    for (RestParameter bparameter : restParameter.getData()) {
-                        if (bparameter.getType().equals("enum")) {
-                            sb.append("        " + getEnumName(bparameter.getTypeClass()) + " " + normalizeNames(getAsCamelCase(bparameter.getName() + "Param")) + " = null;");
-                            sb.append("\n        if (commandOptions." + normalizeNames(getAsCamelCase(bparameter.getName())) + " != null) {\n ");
-                            sb.append("        " + normalizeNames(getAsCamelCase(bparameter.getName() + "Param")) + " = "
-                                    + getEnumName(bparameter.getTypeClass()) + ".valueOf(commandOptions." + normalizeNames(getAsCamelCase(bparameter.getName())) + ");\n");
-                            sb.append("\n        } \n");
-                            //   sb.append(getSwitchEnum(bparameter, normaliceNames(getAsCamelCase(bparameter.getName() + "Param")), getEnumName(bparameter.getTypeClass())));
-                        }
-                    }
-                }
-            }
 
-            sb.append("\n        " + bodyClassName + " " + getAsVariableName(bodyClassName) + " = new " + bodyClassName + "();");
+            String variableName = getAsVariableName(bodyClassName);
+            sb.append("\n        " + bodyClassName + " " + variableName + " = new " + bodyClassName + "();");
             sb.append("\n        if (commandOptions.jsonDataModel) {");
             sb.append("\n            RestResponse<" + getValidResponseNames(restEndpoint.getResponse()) + "> res = new RestResponse<>();");
             sb.append("\n            res.setType(QueryType.VOID);");
-            sb.append("\n            PrintUtils.println(getObjectAsJSON(" + getAsVariableName(bodyClassName) + "));");
+            sb.append("\n            PrintUtils.println(getObjectAsJSON(" + variableName + "));");
             sb.append("\n            return res;");
             sb.append("\n        } else if (commandOptions.jsonFile != null) {");
-            sb.append("\n            " + getAsVariableName(bodyClassName) + " = JacksonUtils.getDefaultObjectMapper()");
-            sb.append("\n                    .readValue(new java.io.File(commandOptions.jsonFile), " + bodyClassName + ".class);");
+            sb.append("\n            ObjectMapper objectMapper = new ObjectMapper();");
```

Comment:
It seems that in a conflictive merge we've lost here this fix:
https://github.com/opencb/opencga/pull/1916